### PR TITLE
refactor(orc8r): replace deprecated errors module

### PR DESCRIPTION
**Pull request by wolfseb**
Replace functions from the github.com/pgk/errors package with
`fmt.Errorf`. Created in pairing with Moritz Huebner.

Signed-off-by: Sebastian Wolf <sebastian.wolf@tngtech.com>

## Summary

This PR implements the proposed solutions of ticket #12632 for the orc8r module.

A few instances of broken rollback error handling next to the changes
in this PR are also fixed.

## Test Plan

Perform Unit tests

## Additional Information

Worked in pairing with @MoritzThomasHuebner 

- [ ] This change is backwards-breaking

### CODE REVIEW

File: orc8r/cloud/go/services/metricsd/servicers/southbound/servicer.go [hunk 1]
```diff
@@ -15,9 +15,9 @@ package southbound
 
 import (
 	"context"
+	"errors"
```
Comment (alexzurbonsen) [inline orc8r/cloud/go/services/metricsd/servicers/southbound/servicer.go:18]: The cases in which std lib "errors" is imported without obvious use in the diff, are due to replacing errors.New from pkg/errors by errors.New from std lib, I assume?

**Review by alexzurbonsen — APPROVED** [review]
Code changes look good to me.

File: orc8r/cloud/go/services/metricsd/servicers/southbound/servicer.go (hunk 1 above)
Comment (wolfseb) [reply orc8r/cloud/go/services/metricsd/servicers/southbound/servicer.go:18]: Yes, exactly. In the same file you should find a removal of github.com/pkg/errors

File: orc8r/cloud/go/services/configurator/mconfig/marshal.go [hunk 2]
```diff
@@ -27,7 +28,7 @@ func MarshalConfigs(configs map[string]proto.Message) (ConfigsByKey, error) {
 	for k, v := range configs {
 		anyVal, err := ptypes.MarshalAny(v)
 		if err != nil {
-			return nil, errors.WithStack(err)
```
Comment (maxhbr) [inline orc8r/cloud/go/services/configurator/mconfig/marshal.go:30]: We have no replacement for `errors.WithStack`?

Comment (sebathomas) [reply orc8r/cloud/go/services/configurator/mconfig/marshal.go:30]: No, you still get the file name and line but no call stack.

**Review by maxhbr — APPROVED** [review]
LGTM

**Pull request merged**